lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -227,6 +227,8 @@ class DocumentImpl extends NodeImpl {
     }
 
     this._contentType = privateData.options.contentType;
+    this._encoding = privateData.options.encoding;
```

Comment:
Mmh, in order to follow the specification, the suggested default encoding is UTF-8 but only "In controlled environments or in environments where the encoding of documents can be prescribed". Browsers don't usually use "UTF-8" as default but "windows-1252" (for an english locale). I think jsdom should follow the browsers behaviour, so should I change the default encoding to "windows-1252" ?